WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -1132,15 +1145,15 @@ nick_the_twister = {
 	chance = {
 		factor = 1
 		modifier = {
-			factor = 1.5
+			factor = 1
 			martial = 17
 		}
 		modifier = {
-			factor = 1.5
+			factor = 1
 			trait = impaler
 		}
 		modifier = {
-			factor = 2
+			factor = 1
```

Comment:
"The Twister" shows up way to common. It's also a rather silly nickname that seems out of place. How many Dark Age or medieval people were called "the Twister" instead of "the twisted"?